src/table/selections-utils.js

```diff
@@ -74,10 +74,12 @@ export function selectCell(cell, selState, selDispatch, evt) {
   const { rowIdx, colIdx, qElemNumber } = cell;
   let selectedRows = [];
 
-  if (!api.isActive()) {
+  if (selState.colIdx === -1) {
```

Comment:
This is always up to date, while there is an bug on isActive (returning false positives)

Reply:
Could you help me understand the colIdx?